src/listener.c

```diff
@@ -399,6 +399,11 @@ void stop_listener(struct listener *l, int lpx, int lpr, int lli)
  */
 void default_add_listener(struct protocol *proto, struct listener *listener)
 {
+	if(global.tune.options & GTUNE_MPTCP)
+		/* check MPTCP option */
+#ifdef linux
```

Comment:
if you are not on Linux, the code will be

```c
if (global.tune.options & GTUNE_MPTCP)
  if (listener->state != LI_INIT)
    return;
```

You need to add the `ifdef` above the `if`